### PR TITLE
Re-Adds Lighters and Dice as Valid Wallet Items

### PR DESCRIPTION
**Pull request by SleepyGemmy**
previous PR (#13076) is fine since it removed stuff like medical items and tools, but i think that lighters and dice should still be valid items since they're basically just for fluff.

also baby's first pr.

edit: also adds keys as valid wallet items.

### CODE REVIEW

**Comment by SleepyGemmy** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]

**Comment by SleepyGemmy** [comment]
> Don't suppose the deluxe jannycart keys could get added too? Always seemed weird to me they can't go into wallets, seeing as they're already tiny and everything.

Done.

File: code/game/objects/items/weapons/storage/wallets.dm [hunk 1]
```diff
@@ -30,7 +32,8 @@
 		/obj/item/device/encryptionkey,
 		/obj/item/fluff,
 		/obj/item/storage/business_card_holder,
-		/obj/item/sample
+		/obj/item/sample,
+		/obj/item/key
```
Comment (Casper3667) [inline code/game/objects/items/weapons/storage/wallets.dm:36]: This should be mentioned in the changelog

**Review by Casper3667 — APPROVED** [review]

File: code/game/objects/items/weapons/storage/wallets.dm (hunk 1 above)
Comment (SleepyGemmy) [reply code/game/objects/items/weapons/storage/wallets.dm:36]: aye aye captain. working on it.

Comment (SleepyGemmy) [reply code/game/objects/items/weapons/storage/wallets.dm:36]: addressed.

**Comment by SleepyGemmy** [comment]
no more items will be added to this PR. let me know if anyone wants anything else added and i will probably open a new PR instead.

**Comment by SleepyGemmy** [comment]
!review

**Review by mikomyazaki — APPROVED** [review]

**Pull request merged**